project/Dependencies.scala

```diff
@@ -41,8 +41,8 @@ object Dependencies {
   val scalaGraphDot =             "org.scala-graph"            %% "graph-dot"                          % "1.11.5"
   val graphvizJava =              "guru.nidi"                  %  "graphviz-java"                      % "0.8.0"
 
-  val catsEffect =                "org.typelevel"              %% "cats-effect"                        % "0.10"
```

Comment:
Is cats-effects 0.10 and 1.X completely binary compatible?

On https://github.com/typelevel/cats-effect/releases they say: "his upgrade breaks binary compatibility with the previous 0.10, so use with care"

We will need to test this upgrade before we merge this.

Reply:
How can this be tested?
All the unit tests for baker still worked. All the unit tests in our application that uses baker still work, and the application still runs.
Is there anything else we can do?

Reply:
We have tested this also with one of our applications and it works fine, so will merge!